README.md

```diff
@@ -1 +1,28 @@
-# node-scanner
\ No newline at end of file
+# Node-Scanner
+Node Version Vulnerability Scanner lightweight CLI tool
+
+This CLI tool is used in conjunction with [`is-my-node-vulnerable`](https://github.com/RafaelGSS/is-my-node-vulnerable), and allows you to properly identify and ignore vulnerabilities found in your installed Node version.
+
+## How to run this tool
+
+After installation, this tool can be run as follows:
+
+`npx node-scanner`
```

Comment:
Assuming we'll publish this as `@vertigis/node-scanner`, it can be run without installation using
```
npx --yes @vertigis/node-scanner
```
or,
```
yarn dlx -p @vertigis/node-scanner
```

Reply:
Fixed